app/main.py

```diff
@@ -1,13 +1,21 @@
 def format_linter_error(error: dict) -> dict:
-    # write your code here
-    pass
+    return {"line": error.get("line_number"),
+            "column": error.get("column_number"),
+            "message": error.get("text"),
+            "name": error.get("code"),
+            "source": "flake8"}
```

Comment:
Place each key on a new line so that there would be no keys on the same line with opening and closing curly braces. It will help to improve the code style and readability